src/backend/utils/misc/guc_gp.c

```
@@ -5717,6 +5729,17 @@ assign_optimizer_log_failure(const char *val, bool assign, GucSource source)
 	return val;
 }
 
+static bool
+assign_pljava_classpath_insecure(bool newval, bool doit, GucSource source)
+{
+	if ( newval == true )
+	{
+		struct config_generic *pljava_cp = find_option("pljava_classpath", false, ERROR);
+		pljava_cp->context = PGC_USERSET;
+	}
+	return true;
+}
+
```

Looks like `pljava_classpatch` can only be set to true and setting it to false is silently ignored?
If that's intentional, might be good to add a code comment explaining why or printing a LOG message.